packages/protocol/test/ACE/NoteRegistry/epochs/201912/NoteRegistry.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable prefer-destructuring */
+/* global artifacts, expect, contract, beforeEach, it, web3:true */
+const { JoinSplitProof, MintProof, note } = require('aztec.js');
+const bn128 = require('@aztec/bn128');
+const { proofs } = require('@aztec/dev-utils');
+const secp256k1 = require('@aztec/secp256k1');
+const BN = require('bn.js');
+
+const ACE = artifacts.require('./ACE');
+const ERC20Mintable = artifacts.require('./ERC20Mintable');
+const JoinSplitValidator = artifacts.require('./JoinSplit');
+const JoinSplitFluidValidator = artifacts.require('./JoinSplitFluid');
+const BaseFactory = artifacts.require('./noteRegistry/epochs/201912/base/FactoryBase201912');
+const AdjustableFactory = artifacts.require('./noteRegistry/epochs/201912/adjustable/FactoryAdjustable201912');
+const BehaviourContract201912 = artifacts.require('./noteRegistry/epochs/201912/Behaviour201912');
+
+const { generateFactoryId } = require('../../../../helpers/Factory');
+const { getNotesForAccount } = require('../../../../helpers/ERC1724');
+
+let ace;
+const aztecAccount = secp256k1.generateAccount();
+const { BOGUS_PROOF, JOIN_SPLIT_PROOF, MINT_PROOF } = proofs;
+const canAdjustSupply = false;
+const canConvert = true;
+let erc20;
+const scalingFactor = new BN(10);
+const tokensTransferred = new BN(100000);
+
+contract('NoteRegistry', (accounts) => {
+    const [aceOwner, owner] = accounts;
+
+    let depositProof;
+    let mintProof;
+    const depositNoteValues = [20, 20];
+    const depositPublicValue = -40;
+
+    beforeEach(async () => {
+        ace = await ACE.new({ from: aceOwner });
+        await ace.setCommonReferenceString(bn128.CRS);
+        ace.setProof(JOIN_SPLIT_PROOF, JoinSplitValidator.address, { from: aceOwner });
+        ace.setProof(MINT_PROOF, JoinSplitFluidValidator.address, { from: aceOwner });
+
+
+        const baseFactory = await BaseFactory.new(ace.address);
+        const adjustableFactory = await AdjustableFactory.new(ace.address);
+
+        await ace.setFactory(generateFactoryId(1, 1, 1), baseFactory.address);
+        await ace.setFactory(generateFactoryId(1, 1, 2), adjustableFactory.address);
+        await ace.setFactory(generateFactoryId(1, 1, 3), adjustableFactory.address);
```

Comment:
same question/comment around the ID/need for the mixed factory ID assigned to the `adjustableFactory.address`